ftw/participation/tests/test_invite.py

```diff
@@ -111,6 +112,28 @@ def test_inviting_a_user_by_userid(self, browser):
         self.assertIn('Hi th=C3=B6re', mail)
 
     @browsing
+    def test_inviting_a_user_by_userid_with_a_different_login_name_is_possible(self, browser):
+        user = create(Builder('user')
+                      .named('Hugo', 'Boss')
+                      .with_userid('hugo.boss')
+                      .with_email('hugo.boss@example.ch'))
+        api.portal.get_tool('acl_users').source_users.updateUser('hugo.boss', 'hugo.boss@example.ch')
```

Comment:
it's not the `userid` and the `email`, it's the `userid` and the `login_name`.

See https://github.com/plone/Products.PlonePAS/blob/master/src/Products/PlonePAS/plugins/property.py#L272